extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/NativeImageProcessor.java

```diff
@@ -302,7 +302,17 @@ void process(
                     org.apache.camel.main.DefaultConfigurationProperties.class,
                     org.apache.camel.main.MainConfigurationProperties.class,
                     org.apache.camel.main.HystrixConfigurationProperties.class,
+                    org.apache.camel.main.Resilience4jConfigurationProperties.class,
```

Comment:
Shouldn't Resilience4jConfigurationProperties be listed below where other `org.apache.camel.main.Resilience4j*` classes are listed?

Reply:
the other are form different packages unless you means some other location

Reply:
Yes, I have not noticed they are from a different package, but I rather meant the same TODO applies to Resilience4jConfigurationProperties

Reply:
ah yeah